Lib/Src/CThermistor.cpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file CThermistor.cpp
+ */
+
+/*
+ *  Created on: Feb 15, 2022
+ *      Author: niuslar
+ */
+
+#include "CThermistor.h"
+#include "math.h"
+
+#define MIN_VOLT_RANGE     (0.57)
+#define MAX_VOLT_RANGE     (2.67)
```

Comment:
Min and Max values should probably be a setting that you can give to the thermistor either as part of constructor or as a separate method. Other thermistors will have different limits, so no need to hardcode the values.